server/pbench/bin/pbench-report-status

```diff
@@ -70,15 +84,12 @@ if [ ! -z "$report_using_mail" -a "$report_using_mail" != "no" ] ;then
     exit $status
```

Comment:
Can we put into the mail subject the timestamp value as well, something like, `$name status report at $timestamp`?

Reply:
OK